packages/chronos/chronos-lib/src/main/java/com/daimler/data/controller/ForecastController.java

```diff
@@ -1079,7 +1079,7 @@ public ResponseEntity<ForecastRunResponseVO> createForecastRun(@ApiParam(value =
     		@ApiParam(value = "Any number greater than 1", required=true) @RequestParam(value="forecastHorizon", required=true)  BigDecimal forecastHorizon,
     		@ApiParam(value = "The file to upload.") @Valid @RequestPart(value="file", required=false) MultipartFile file,
     		@ApiParam(value = "path of file in minio system, if not giving file in request part") @RequestParam(value="savedInputPath", required=false)  String savedInputPath,
-    		@ApiParam(value = "flag whether to save file in request part to storage bucket for further runs") @RequestParam(value="saveRequestPart", required=false)  Boolean saveRequestPart,
+			@ApiParam(value = "flag whether to save file in request part to storage bucket for further runs", defaultValue="false") @RequestParam(value="saveRequestPart", required=false)  Boolean saveRequestPart,
```

Comment:
bug in swagger ui 3 spring fox, wouldnt still show on ui that default value is false. Can still remain.